### PR TITLE
USHIFT-2807: Add selected rpm-ostree scenario tests to bootc on CentOS

### PR DESCRIPTION
**Pull request by ggiguash**

### CODE REVIEW

File: test/bin/pyutils/build_bootc_images.py [hunk 1]
```diff
@@ -199,8 +200,8 @@ def main():
         if args.group_dir:
             process_containerfiles(args.group_dir, args.dry_run)
             return
-        # Process a layer with group directories
-        for item in os.listdir(args.layer_dir):
+        # Process layer directory contents sorted by length and then alphabetically
+        for item in sorted(os.listdir(args.layer_dir), key=lambda i: (len(i), i)):
```
Comment (pmtk) [inline test/bin/pyutils/build_bootc_images.py:204]: It doesn't help with parallelization of the process, right? Similar to what we have now, seems like `cos9-bootc-source-optionals.containerfile` is material for `group2`

Comment (ggiguash) [reply test/bin/pyutils/build_bootc_images.py:204]: Very true. We have a separate issue on the parallelization [USHIFT-2929](https://issues.redhat.com/browse/USHIFT-2929). I think this is where we should do the restructuring and test the time gain, etc.

**Pull request merged**